tico/serialize/circle_serializer.py

```diff
@@ -213,6 +214,7 @@ def build_circle(edge_program: ExportedProgram) -> bytes:
             logger.debug(f"call_function: {node.name} ({opcode}) Op exported.")
 
     # Register subgraph
+    finalise_tensor_names(graph)
```

Comment:
`node.meta.get("nn_module_stack")` will be kept until this point. Why can't we can update node names directly?